tensorflow/examples/learn/examples_test.sh

```diff
@@ -28,20 +28,18 @@ then
   DIR="$DIR"/"$TEST_WORKSPACE"
 fi
 
-SKFLOW_EXAMPLE_BASE_DIR=$DIR/tensorflow/examples/learn
+TFLEARN_EXAMPLE_BASE_DIR=$DIR/tensorflow/examples/learn
```

Comment:
Well in my opinion `TF_LEARN_EXAMPLE` sounds like some TensorFlow examples you can learn. But anyways, I don't think users care about this script so let's just go with this at this moment.